analyzers/src/SonarAnalyzer.CFG/LiveVariableAnalysis/RoslynLiveVariableAnalysis.cs

```diff
@@ -138,6 +138,10 @@ private void BuildBranches(BasicBlock block)
                 AddPredecessorsOutsideRegion(finallyBlock);
             }
         }
+        if (block.IsEnclosedIn(ControlFlowRegionKind.Catch) && block.Successors.FirstOrDefault(x => x.Semantics == ControlFlowBranchSemantics.Rethrow) is { })
```

Comment:
```suggestion
        if (block.IsEnclosedIn(ControlFlowRegionKind.Catch) && block.Successors.FirstOrDefault(x => x.Semantics == ControlFlowBranchSemantics.Rethrow) is not null)
```

It's probably an FN for T0007.